pages/index.tsx

```diff
@@ -101,7 +98,9 @@ const Home: NextPage<{ adobeAnalyticsUrl: string }> = ({
             <li dangerouslySetInnerHTML={{ __html: tsln.maritalStatusText }} />
             <li dangerouslySetInnerHTML={{ __html: tsln.partnerText }} />
           </ul>
-
+          <p className="xs:mt-0 sm:mt-8 xs:pr-3 w-full sm:w-[73%]">
```

Comment:
73% ???

Reply:
If you look at the figma mockup you will know, it wraps the last word to the next line.

Reply:
you could do the same by adding  &nbsp;  instead